dotnet/src/Agents/Abstractions/AgentChat.cs

```diff
@@ -211,6 +238,8 @@ await foreach (ChatMessageContent message in channel.InvokeAsync(agent, cancella
                     .Where(kvp => kvp.Value != channel)
                     .Select(kvp => new ChannelReference(kvp.Value, kvp.Key));
             this._broadcastQueue.Enqueue(channelRefs, messages);
+
+            this.Logger.LogInformation("[{MethodName}] Invoked agent {AgentType}: {AgentId}", nameof(InvokeAgentAsync), agent.GetType(), agent.Id);
```

Comment:
Same here, information about the beginning of operation (i.e. `invoking`) has level `Debug` while completion (i.e. `invoked`) has level `Information`. What is the reason of that?

Reply:
Same reason.